### ISSUE TITLE
panic at membersAt func due to index out of range

### ISSUE THREAD

**Issue by devtomnl** [dusk-network/dusk-blockchain#574]
```
panic: runtime error: index out of range [255] with length 255
goroutine 152195 [running]:
github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee.(*Handler).membersAt(0xc001131480, 0xd664ff, 0x0)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee/handler.go:88 +0xdd
github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee.(*Handler).Committee(0xc001131480, 0x5d2, 0xff, 0x40, 0x0, 0x0, 0x0, 0x0)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee/handler.go:52 +0x38
github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee.(*Handler).IsMember(0xc001131480, 0xc00003c480, 0x81, 0x81, 0x5d2, 0xff, 0x40, 0x20)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee/handler.go:42 +0x4f
github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee.(*Handler).AmMember(...)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/committee/handler.go:36
github.com/dusk-network/dusk-blockchain/pkg/core/consensus/reduction.(*Handler).AmMember(...)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/reduction/handler.go:35
github.com/dusk-network/dusk-blockchain/pkg/core/consensus/reduction/secondstep.(*Reducer).CollectStepVotes(0xc000f0c870, 0x10554c0, 0xc00139aa20, 0x977811, 0xc0013281c0)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/reduction/secondstep/reducer.go:190 +0x336
github.com/dusk-network/dusk-blockchain/pkg/core/consensus.(*SimpleListener).NotifyPayload(0xc000c3ce70, 0x10554c0, 0xc00139aa20, 0x10554c0, 0xc00139aa20)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/component.go:105 +0x3d
github.com/dusk-network/dusk-blockchain/pkg/core/consensus.(*roundStore).Dispatch(0xc000ea2940, 0x10650e0, 0xc0012d2e60)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/round.go:122 +0x2d8
github.com/dusk-network/dusk-blockchain/pkg/core/consensus.(*Coordinator).CollectEvent(0xc0002fc580, 0x10650e0, 0xc0012d2e60, 0xc000144250, 0x1)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/core/consensus/round.go:356 +0x74c
github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/eventbus.(*CallbackListener).Notify(0xc000227940, 0x10650e0, 0xc0012d2e60, 0x1, 0x1)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/eventbus/listener.go:32 +0x9d
github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/eventbus.(*multiListener).Forward(0xc0001ef6c0, 0x20, 0x10650e0, 0xc0012d2e60)
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/eventbus/listener.go:180 +0x13c
created by github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/eventbus.(*EventBus).Publish
	/opt/gocode/src/github.com/dusk-network/dusk-blockchain/pkg/util/nativeutils/eventbus/publisher.go:26 +0x244
```

**Comment by autholykos** [comment]
Rather than fix this problem, we should make sure that when it panics, it provides the right information about what is going on

**Comment by autholykos** [comment]
Closed via #631
